### ISSUE TITLE
reST syntax is incorrect in code-dev.rst

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8416]
|[<img alt="chellygel's avatar" src="https://avatars.githubusercontent.com/u/634036?s=50" width="50" height="50">](https://github.com/chellygel)| @chellygel reported|
|-|-|
|Trac ID|trac#8416|
|Type|defect|
|Created|2016-06-03 23:04:35Z|
|Branch|https://github.com/twisted/twisted/tree/restfix-8416|

reST syntax in code-dev.rst only uses single backticks instead of double backticks.

<details><summary>Searchable metadata</summary>

```
trac-id__8416 8416
type__defect defect
reporter__chellygel chellygel
priority__lowest lowest
milestone__None None
branch__restfix_8416 restfix-8416
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__None None
time__1464995075087293 1464995075087293
changetime__1474198569965514 1474198569965514
version__None None
owner__Glyph__glyph_____ Glyph <glyph@...>
cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="chellygel's avatar" src="https://avatars.githubusercontent.com/u/634036?s=50" width="50" height="50">](https://github.com/chellygel)<a name="note_2"></a>|@chellygel commented|
|-|-|

https://github.com/twisted/twisted/pull/132

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @glyph<br>@glyph set status to `closed`|
|-|-|

In changeset 408b19c0467f0746deec3b16ce9ae23088f8f2ae
```
#!CommitTicketReference repository="" revision="408b19c0467f0746deec3b16ce9ae23088f8f2ae"
Merge pull request #132 from alex/patch-1

Author: alex

Reviewer: glyph

Fixes: #8416

Fixed some markup to avoid ugly artifacts.
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing the review tag from closed tickets
